fili-core/src/main/java/com/yahoo/bard/webservice/druid/model/virtualcolumns/VirtualColumn.java

```diff
@@ -0,0 +1,47 @@
+// Copyright 2016 Yahoo Inc.
+// Licensed under the terms of the Apache license. Please see LICENSE.md file distributed with this work for terms.
+package com.yahoo.bard.webservice.druid.model.virtualcolumns;
+
+import com.yahoo.bard.webservice.druid.model.VirtualColumnType;
+
+/**
+ * This is marker a interface to reference Virtual Columns.
+ */
+public interface VirtualColumn {
+
+    /**
+     * Get the name of the virtual column in the response.
+     *
+     * @return the name of the virtual column in the response
+     */
+    String getName();
+
+    /**
+     * Get the type of the virtual column in the response.
+     *
+     * @return the type of the virtual column in the response
+     */
+    VirtualColumnType getType();
+
+    /**
+     * Get the expression of the virtual column in the response.
```

Comment:
I suspect we should pull this from the interface, since the expression virtual column may be the only one that uses it.
Obviously I'm speculating since there's exactly one.  But I like your instinct to make a type to avoid refactoring interfaces later.